### DISCUSSION TITLE
Virus positive

### DISCUSSION THREAD

**Discussion by ghost** [LesFerch/WinSetView#64] [General]
When i downloaded file from Your GithHub repository Defender detect virus:

![image](https://user-images.githubusercontent.com/129328996/228649503-b7ae7835-cf6e-4c02-be73-3a25fd26a39c.png)

Please, correct your file. Thanks.

**Comment by LesFerch** [comment]
Nothing to correct. It's a false positive. I have not seen a false positive with Windows Defender in two years. Is this perhaps BitDefender?
